### PR TITLE
Hyper link CTA with secondary style

### PR DESCRIPTION
**Pull request by blackfalcon**
# Alaska Airlines Pull Request

This PR includes a minor refactor of how property-based styles are applied to the custom element for customization as well as new support for a secondary style CTA.

**Fixes:** #5 

## Summary:

The first part of this PR includes a refactor of the process for adding CSS classes to the custom element by using the element's API. The idea is to reduce this to a single function as not to have a long list of functions in the `class` attribute. 

The second part of this PR is to add support for a secondary button style for the CTA. This is to model the same approach for creating a secondary style button with the ods-button element. 

## Type of change:

Please delete options that are not relevant.

- [x] New capability 
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate) 

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._ 

**Thank you for your submission!**<br>
-- Orion Design System Team

### CODE REVIEW

File: src/ods-hyperlink.js [hunk 1]
```diff
@@ -147,7 +149,7 @@ class OdsHyperlink extends LitElement {
         ?download="${this.download}"
         role="${ifDefined(this.role === 'button' || this.role === 'tab' ? this.role : undefined)}"
         rel="${ifDefined(this.target || this.rel ? this.getReltype(this.target, this.rel) : undefined)}"
-        class="${this.getAnchortype(this.role, this.href)} ${this.getContext(this.inline)} ${this.getTheme(this.darktheme)} ${this.getCta(this.cta)} ${this.getTabState(this.tabisactive)}"
+        class="${this.getStyles(this.role, this.darktheme, this.inline, this.cta, this.buttontype)} ${this.getTabState(this.tabisactive)}"
```
Comment (geoffrey1218) [inline src/ods-hyperlink.js]: Do you need to pass these arguments here if you're accessing them all with `this` in `getStyles`?

```suggestion
        class="${this.getStyles()} ${this.getTabState(this.tabisactive)}"
```

Comment (blackfalcon) [reply src/ods-hyperlink.js]: Updated ... thanks 👍

**Comment by blackfalcon** [comment]
All work per this PR is completed. Ready for REVIEW.

**Comment by ecommerceUXdesign** [comment]
PR will not pass checks until ods-button PR is approved and this repo is updated with the new package reference.

**Pull request merged**